src/screens/authentication/UsernameRequest/index.js

```diff
@@ -19,70 +42,66 @@ const SubmitIndicator = ({ submitting }) => (
     />
 )
 
-const Error = ({ error }) => (
-  error && (
-    <Body
-      secondary
-      style={styles.error}
-    >
-      {error}
-    </Body>
-  )
+const SubmitButton = ({ disabled, onPress, error }) => (
+  <PrimaryButton
+    onPress={onPress}
+    style={error ? { marginTop: verticalScale(20) } : { marginTop: verticalScale(40) }}
+    disabled={error ? true : disabled}
```

Comment:
This is equivalent to `error || disabled`

Reply:
I tried it  that way but it caused a warning because error is a string and the disable prop only takes booleans

Reply:
You can convert it to Boolean: `Boolean(error || disabled)`